netci.groovy

```diff
@@ -137,6 +137,11 @@ branchList.each { branchName ->
             // Add the unit test results
             Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
 
+            // Unix runs take more than 2 hours to run, so we set the timeout to be longer.
+            if (os == 'Ubuntu' || os == 'OSX') {
+                Utilities.setJobTimeout(newJob, 240)
+            }
```

Comment:
The problem is that there are a bunch of different versions of Windows that this could take, like: "Windows 10" "Windows 7" or "Windows_NT". I added this since this is only required very specifically to Ubuntu and OSX since we are using .NetCore MSBuild for those two platforms.